packages/bvaughn-architecture-demo/src/suspense/PointsBoundingTimeCache.ts

```diff
@@ -0,0 +1,14 @@
+import { getPointsBoundingTimeResult } from "@replayio/protocol";
+
+import { ReplayClientInterface } from "shared/client/types";
+
+import { createGenericCache2 } from "./createGenericCache";
+
+export const {
```

Comment:
I think this should probably go into the `PointsCache` module. That's where the call to `getPointsBoundingTime` is and this seems basically like a variant of that.